tensorboard/components/tf_markdown_view/tf-markdown-view.ts

```diff
@@ -46,6 +46,9 @@ class TfMarkdownView extends LegacyElementMixin(PolymerElement) {
       #markdown > p:last-child {
         margin-bottom: 0.3em;
       }
+      #markdown p {
+        white-space: break-spaces;
```

Comment:
Mind adding a comment or just a link to the bug (imo its ok that the bug is internal only).

Reply:
Done.